examples/tenant_cluster/README.md

```diff
@@ -0,0 +1,46 @@
+# atlas-basic - free tier cluster (tenant)
+
+This example shows how the atlas-basic module can be used to create a free-tier cluster (TENANT).
```

Comment:
Same comment as before: please use active voice. You can use -- see the very first comment for an example of the rewrite please.